### DISCUSSION TITLE
How to debug source code？

### DISCUSSION THREAD

**Discussion by huifer** [tauri-apps/tauri#11911] [Q&A]
I want to know how to debug after downloading the tauri source code program. Especially build.rs and some other stuff. I haven't found any relevant documentation yet. If there is any, please send it to me.

**Comment by FabianLars** [comment]
Debugging build scripts (build.rs) is pretty annoying because of issues like this https://github.com/rust-lang/cargo/issues/985 - nowadays `cargo build -vv` at least helps somewhat.

But really, there's nothing reallyyy special about tauri's source so if you search for rust debugging tools the search results should apply to tauri's source too.
